roles/etcd_common/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 r_etcd_common_etcd_runtime: "docker"
 
 # etcd server vars
-etcd_conf_dir: "{{ '/etc/etcd' if r_etcd_common_etcd_runtime != 'runc' else '/var/lib/etcd/etcd.etcd/etc'  }}"
+etcd_conf_dir: "{{ '/etc/etcd' if not openshift.common.is_etcd_system_container else '/var/lib/etcd/etcd.etcd/etc'  }}"
```

Comment:
That should be identical. What play calls the `etcd_common` role? There is `openshift.common.etcd_runtime` that is set to `run` iff `openshift.common.is_etcd_system_container` is true.